src/main/java/com/backendoori/ootw/weather/util/client/ForecastApi.java

```diff
@@ -0,0 +1,26 @@
+package com.backendoori.ootw.weather.util.client;
+
+import com.backendoori.ootw.weather.domain.ForecastResult;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@FeignClient(value = "forecast-api", url = "http://apis.data.go.kr/1360000/VilageFcstInfoService_2.0")
+public interface ForecastApi {
+
+    @GetMapping(value = "/getVilageFcst")
+    ForecastResult getVillageForecast(@RequestParam String serviceKey, @RequestParam int numOfRows,
+                                      @RequestParam int pageNo, @RequestParam String dataType,
+                                      @RequestParam(name = "base_date") String baseDate,
+                                      @RequestParam(name = "base_time") String baseTime, @RequestParam int nx,
+                                      @RequestParam int ny
+    );
```

Comment:
이런 식으로 라인을 나누는 것도 좋을 것 같아요

```suggestion
    @GetMapping(value = "/getVilageFcst")
    ForecastResult getVillageForecast(
        @RequestParam String serviceKey,
        @RequestParam int numOfRows,
        @RequestParam int pageNo,
        @RequestParam String dataType,
        @RequestParam(name = "base_date") String baseDate,
        @RequestParam(name = "base_time") String baseTime, 
        @RequestParam int nx,
        @RequestParam int ny
    );
```